app/routes/user/endorsements.server.ts

```diff
@@ -67,14 +68,7 @@ export class EndorsementsServer {
       )
 
     const escapedEndorserString = endorserSub.replaceAll('"', '\\"')
```

Comment:
Move the escaping not the `#getCognitoUsernameForSub` method.

Reply:
Where should the escaping get moved to? It is used a few lines lower in addition to the get method

Reply:
I'm OK with repeating it here and also inside `#getCognitoUsernameForSub`. What I want to avoid is future bugs related to confusion about which functions expect sub itself versus which functions expect escaped sub.